mac

```diff
@@ -181,16 +181,36 @@ install_asdf_language() {
   fi
 }
 
+install_asdf_language_version() {
+  local language="$1"
+  local version="$2"
+
+  if ! asdf list "$language" | grep -Fq "$version"; then
+    asdf install "$language" "$version"
+    asdf global "$language" "$version"
+  fi
+}
+
+start_postgres_server() {
+  if ! pg_ctl status | grep -Fq "server is running"; then
+    pg_ctl start
+  fi
+}
+
 fancy_echo "Installing latest Ruby..."
-install_asdf_language "ruby"
+install_asdf_language_latest "ruby"
```

Comment:
Does this always make it the most recent ruby? Do we want that?

Reply:
Would we just overwrite it when necessary?

Reply:
Seems like latest or not shouldn't matter too much because it will be overwritten by most apps. I can see having the newest ruby being an advantage (all the newest things), but also a disadvantage (some CLI ruby gem doesn't play nice, testing with a newer ruby than we have access to).

Any specific reason behind this change? Does it make other parts of the setup work better?

Reply:
It looks like this actually has more to do with renaming the function, not changing the logic.